modules/contact/src/constraints/ComputeDynamicFrictionalForceLMMechanicalContact.C

```diff
@@ -277,6 +294,10 @@ ComputeDynamicFrictionalForceLMMechanicalContact::enforceConstraintOnDof3d(
   const Real c = _normalize_c ? _c / *_normalization_ptr : _c;
   const Real c_t = _normalize_c ? _c_t / *_normalization_ptr : _c_t;
 
+  // Compute the friction coefficient (constant or function)
+  ADReal mu_ad = 0.0;
+  computeFrictionValue(mu_ad, contact_pressure, *tangential_vel[0], *tangential_vel[1]);
```

Comment:
This is fine, but could just have `computeFrictionValue` return an `ADReal` instead of being `void`